socorro/unittest/processor/rules/test_general.py

```diff
@@ -110,7 +111,31 @@
 )
 
 
-class TestDeNullRule(object):
+class TestDeNoneRule:
+    @pytest.mark.parametrize(
+        "raw_crash, expected",
+        [
+            ({}, {}),
+            ({"foo": "bar"}, {"foo": "bar"}),
+            ({"foo": None}, {}),
+            ({"foo": "bar", "baz": None}, {"foo": "bar"}),
+        ],
+    )
+    def test_denone(self, raw_crash, expected):
+        rule = DeNoneRule()
+        rule.action(raw_crash, None, {}, {})
+        assert raw_crash == expected
+
+    def test_denone_with_dotdict(self):
+        raw_crash = DotDict({"foo": "bar", "baz": None})
+        expected = DotDict({"foo": "bar"})
+
+        rule = DeNoneRule()
+        rule.action(raw_crash, None, {}, {})
+        assert raw_crash == expected
```

Comment:
I kept the `DotDict` test separate because at some point, we're going to remove `DotDict` usage.